server/issue.go

```diff
@@ -1141,13 +1141,11 @@ func (s *ConnectionSettings) ShouldReceiveNotification(role string) bool {
 }
 
 func (p *Plugin) fetchConnectedUserFromAccount(account map[string]string, instance Instance) (Client, *Connection, error) {
-	var mattermostUserID types.ID
-	var err error
+	accountKey := account["Name"]
 	if account["AccountID"] != "" {
```

Comment:
why not do the same for account["AccountID"] like you did for account["Name"] means making a variable

Reply:
I think this improve more readability, also It is suggested in one of the review comments.